src/additional/ishexcolor.js

```diff
@@ -0,0 +1,3 @@
+$.validator.addMethod( "ishexcolor", function( value, element ) {
```

Comment:
Can you change the name to `hexcolor` in order to be consistent with other additional methods?
Also, I would like the file to be renamed to `hexcolor` as well.